python/PyQt6/core/auto_generated/symbology/qgslinesymbollayer.sip.in

```diff
@@ -950,6 +950,25 @@ direction is averaged when calculating individual symbol angles.
 .. seealso:: :py:func:`averageAngleUnit`
 %End
 
+
+    void setBlankAreasUnit( Qgis::RenderUnit unit );
```

Comment:
I'm not a big fan of "blank areas" to describe this -- could we use something like "blank substrings" or "blank segments" or "blanked distances"/"suppressed distances" instead? (It's the "area" part I think is misleading, since we're working with distances here, not areas)